src/main/webapp/assets/script.js

```diff
@@ -254,7 +333,8 @@ function addPosts(posts) {
   const allPosts = document.getElementById('all-posts');
   posts.forEach((post) => {
     const titleText = post.organizationName + ' @ ' + post.location.name;
-    const subtitleText = post.foodType + ' | ' + post.eventStartTime + '-' + post.eventEndTime;
+    const subtitleText = post.foodType + ' | ' + post.eventStartTime.toLocaleTimeString('en-US') +
+      '-' + post.eventEndTime.toLocaleTimeString('en-US');
     const descriptionText = post.description;
 
     // Create card.
```

Comment:
Would it be possible to have a separate method just for creating cards since it's a lot of adding attributes for one function?

Reply:
I didn't originally write this method (just updated the formatting of the time strings since `Date` manipulation was part of this PR), but I personally don't think that moving the card-creation logic to another method would be worthwhile. The main reason the function is so long is mostly just a side effect of the fact that DOM manipulation through JS is pretty verbose (it takes 4 lines to create a simple element with content). Additionally, the logic of creating a single `post` card outside of the context of creating all recent `post` cards isn't likely to be reused anywhere else in our app. Let me know if you disagree though, and we can figure it out!

Reply:
Just a style preference so no worries if you prefer this!